src/vl-map-vector-layer.js

```diff
@@ -52,6 +52,7 @@ export class VlMapVectorLayer extends VlMapLayer {
       this._styles = [];
       this._layer.setStyle(style);
     }
+    this.dispatchEvent(new CustomEvent('style-changed', {bubbles: true, composed: true, detail: {style: style}}));
```

Comment:
Misschien ook best ergens documenteren en/of de naam van de event in een constante gieten?

Reply:
Done